scripts/get_l10ns

```diff
@@ -6,4 +6,6 @@ if [ "$1" == "" ]; then
 fi
 
 echo "Downloading l10ns from Crowdin."
-python source/common/res/features/l10n/init.py $1
+cd src/extension/legacy/features/l10n
```

Comment:
I like to use `pushd` and `popd` in these scenarios so you always end up where you started.